tests/PhpSpreadsheetTests/Writer/Xlsx/Issue3443Test.php

```diff
@@ -37,6 +37,7 @@ public function testNonDefaultAlign(): void
         $rsheet = $reloadedSpreadsheet->getActiveSheet();
         $expected1 = [
             'horizontal' => 'center',
+            'justifyLastLine' => false,
```

Comment:
This will probably be affected by the change I suggested above to Reader/Xlsx/Styles.